auth/src/main/java/com/firebase/ui/auth/provider/Provider.java

```diff
@@ -5,15 +5,10 @@
 import android.content.Intent;
 import android.support.annotation.LayoutRes;
 
-import com.google.firebase.auth.GoogleAuthProvider;
-
 public interface Provider {
     /** Retrieves the name of the IDP, for display on-screen. */
     String getName(Context context);
 
-    /** Retrieves the id of the IDP, e.g. {@link GoogleAuthProvider#PROVIDER_ID}. */
-    String getProviderId();
```

Comment:
We never use this anywhere which makes sense since most of the time we need the provider id to figure out which class to instantiate. Thus, we would already have the provider id by the time we get an instance of a `Provider`.